src/actions/firebase/firebase.ts

```diff
@@ -121,14 +122,12 @@ export class FirebaseAction extends Hub.Action {
     })
   }
 
-  async sendMessageToDevice(deviceId: string,
+  async sendMessageToDevice(message: firebaseAdmin.messaging.TokenMessage,
                             webhookId: string | undefined,
-                            payload: firebaseAdmin.messaging.MessagingPayload,
-                            options?: firebaseAdmin.messaging.MessagingOptions,
-                            ): Promise<any> {
+  ): Promise<any> {
     return new Promise<void>((resolve, reject) => {
       FirebaseAction.setFirebaseClient()
-      firebaseAdmin.messaging().sendToDevice(deviceId, payload, options)
+      firebaseAdmin.messaging().send(message)
```

Comment:
Does `send` no longer need options?  Just trying to understand the purpose of `this.notificationOptions`

Reply:
From Phillip:
> looks like those notification options don't exist anymore